content/pt/index.md

```diff
@@ -1,125 +1,125 @@
 ---
 template: blank
-title: 'The Intuitive Vue Framework'
-description: 'Build your next Vue.js application with confidence using Nuxt. An open source framework making web development simple and powerful.'
+title: 'O framework Intuitivo de Vue'
+description: 'Construa a sua próxima aplicação em Vue.js usando o Nuxt. Um framework de código-aberto tornando o desenvolvimento web simples e poderoso.'
 bgClass: 'bg-white'
 ---
 
 ::home-hero
 #title
-The Intuitive Vue<br>Framework
+O framework<br>Intuitivo de Vue
 
 #description
-Build your next Vue.js application with confidence using Nuxt.<br class="hidden sm:block" /> An open source framework making web development simple and powerful.
+Construa a sua próxima aplicação em Vue.js usando o Nuxt.<br class="hidden sm:block" /> Um framework de código-aberto tornando o desenvolvimento web simples e poderoso.
 
 #primary-button
-:app-button[39K+ GitHub stars]{ href="https://github.com/nuxt/nuxt.js" icon="IconGitHub" extraClass="text-white bg-sky-black hover:bg-sky-darker leading-4" }
+:app-button[39K+ estrelas no GitHub]{ href="https://github.com/nuxt/nuxt.js" icon="IconGitHub" extraClass="text-white bg-sky-black hover:bg-sky-darker leading-4" }
 
 #secondary-button
-:app-button[Get started]{ to="/docs/get-started/installation" }
+:app-button[Começar]{ to="/docs/get-started/installation" }
 ::
 
 ::home-learn-master
 ---
-category: Learn
+category: Aprender
```

Comment:
@smarroufin are those supposed to be tranlsated?

Reply:
I was using French version to inform myself what is supposed to be translated.